source/Octopus.Tentacle.Tests.Integration/Support/TentacleFetchers/LinuxTentacleFetcher.cs

```diff
@@ -3,6 +3,7 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
+using Octopus.Tentacle.CommonTestUtils;
```

Comment:
Mind pulling out the move/refactoring to a different PR?

Reply:
I'm actually going to recreate this branch from scratch using the test code folder

Reply:
I've split this into 2 PR's. One just moving test code, and one will be adding the new tests
https://github.com/OctopusDeploy/OctopusTentacle/pull/887